ProductTree/ViewModels/ProductTreeRows/ActualFiniteStateRowViewModel.cs

```diff
@@ -56,6 +61,7 @@ public ActualFiniteStateRowViewModel(ActualFiniteState actualFiniteState, ISessi
             : base(actualFiniteState, session, containerViewModel)
         {
             this.IsPublishable = false;
+            this.IsDefault = actualFiniteState.IsDefault;
```

Comment:
This is not entirely correct I think. The row might not react if the state is no longer default. Please verify.